lib/services/nodes-api-service.js

```diff
@@ -316,6 +314,9 @@ function nodeApiServiceFactory(
                 };
                 return workflowApiService.createAndRunGraph(configuration);
             }
+            else if(node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added');
```

Comment:
We can do it for all nodes, that's what it did do, I was just changing it to actually meet the requirements of the original work item but keeping it for all nodes is fine. If we want that I can just close the PR